plugins/obs-vst/obs-vst.cpp

```diff
@@ -130,14 +128,20 @@ static void vst_update(void *data, obs_data_t *settings)
 	}
 	vstPlugin->loadEffectFromPath(std::string(path));
 
-	std::string hash = getFileMD5(path);
-	const char *chunkHash = obs_data_get_string(settings, "chunk_hash");
+	const int32_t hash = vstPlugin->getUniqueID();
+
+	const int32_t chunkHash = obs_data_get_int(settings, "chunk_hash");
```

Comment:
As far as I can tell, the intent here is to try to load `chunk_hash` as an int32_t via `obs_data_get_int`, which is expected to return `0` if the data type does not match. The expectation is that `chunk_hash` will be `0` if it's actually stored as a string, which is expected to be an MD5 hash. If so, `chunkHashesMatch` will be `false`, and the fallback path will be taken. Is that an accurate summary of the intent?

Has this actually been tested? I'm a little concerned that storing data in `chunk_hash` that is not actually a hash could lead to confusion down the road and could make it more difficult to discern if a VST has a uniqueID.

Reply:
yes, that is the intention. I have tested locally both scenarios, where the session was saved with string, and with int. In both cases it loaded the chunk correctly.